qemu/tests/cfg/ovs_host_vlan.cfg

```diff
@@ -0,0 +1,43 @@
+- ovs_host_vlan:
+    type = ovs_host_vlan
+    private_vlan_bridge = "ovs_private_br"
+    only Linux
+    bridge_ip = 192.168.10.1/24
+    ovs_br_create_cmd = "ovs-vsctl br-exists ${private_vlan_bridge} || ovs-vsctl add-br ${private_vlan_bridge};"
+    ovs_br_create_cmd += "ifconfig ${private_vlan_bridge} ${bridge_ip} up"
+    ovs_br_remove_cmd = "ifconfig ${private_vlan_bridge} down; ovs-vsctl del-br ${private_vlan_bridge}"
+    qemu_sandbox = off
+    vms = "vm1 vm2 vm3 vm4"
+    image_snapshot = yes
+    start_vm = no
+    ip_nic1_vm1 = 192.168.10.11
+    ip_nic1_vm2 = 192.168.10.12
+    ip_nic1_vm3 = 192.168.10.21
+    ip_nic1_vm4 = 192.168.10.22
+    nic_ifname_vm1 = ovs_vm1
+    nic_ifname_vm2 = ovs_vm2
+    nic_ifname_vm3 = ovs_vm3
+    nic_ifname_vm4 = ovs_vm4
```

Comment:
Should these 4 options be deleted? Looks like they are not used in the testings. The ifname of the nic will be generated automatically as T$_nic_index_$random_str.